### DISCUSSION TITLE
How to plot to a network drive?

### DISCUSSION THREAD

**Discussion by 8PotatoChip8** [guydavis/machinaris#513] [Q&A]
I am looking to plot on a computer that will then transfer the finished plots to a network drive, I am unsure as to how to add network drives to the Machinaris worker creator. Thanks

**Comment by guydavis** [comment]
Hi, what OS are you running Machinaris on?

EDIT: Just saw another discussion from you where you report running on Windows.  Please see the [wiki](https://github.com/guydavis/machinaris/wiki/Windows#network-share-access) for mounting network shares directly inside the Machinaris container to avoid the Docker limitations of mounting the share on Windows.

**Reply by 8PotatoChip8** [reply]
Right! I understand that, but the "New Worker" window (which you access by hitting "New Worker" in the workers tab) does not seem to have the ability to select whether a drive is local or remote whereas when setting up Machinaris it allowed for local or remote drive. I also understand it it possible to create your own docker-compose.yml file so you can make it work, which I have done. But I am unsure as to whether I have done it correct, I also think it would be very helpful if the remote and local drive option was added to the "New Worker" tab aswell, especially helpful for beginners.
